conf/config.yaml.sample

```diff
@@ -17,6 +17,7 @@ cortx:
     consul:
       endpoints:
         - tcp://consul-server.default.svc.cluster.local:8301
+        - http://consul-server.default.svc.cluster.local:8500
```

Comment:
Should this be K:V instead of an array?

Reply:
no. This must be http: which is a standard way of accessing. Components need to parse it and convert it to consul://consul......

Reply:
Why are we changing protocol from TCP to HTTP again this will not be consistent with kafka definition:
`tcp://kafka.default.svc.cluster.local:9092`
Either keep this as TCP or change this to CONSUL 

Also please change definition for provisioner test framework as well:
https://github.com/Seagate/cortx-prvsnr/blob/kubernetes/test/deploy/kubernetes/solution-config/config.yaml

Test thoroughly to ensure deployment works fine...

Once done please raise ticket for Cortx-Services team to address change in their repo:
https://github.com/Seagate/cortx-k8s/blob/stable/k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/config-template.yaml

Accordingly, lets update confluence as well:
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/611615649/CORTX+Configuration+Parameters

Reply:
@tanujashinde0405 Please ensure we address above comment (to be consistent across all repos and documents)...

Reply:
Endpoints and Protocols are defined by the Services. We only have to represent it here. 
Consul provides two kinds of endpoints (1) TCP mainly for Consul Agent to be able to talk to Consul Server (Proxy Pattern). (2) HTTP for Python based Consul Clients (Client-Server Pattern). We use both kinds of endpoints in our stack.